lib/pbench/test/unit/server/query_apis/test_namespace_and_rows.py

```diff
@@ -327,7 +327,7 @@ def test_query(
                 },
             },
         }
-        index = self.build_index_from_metadata()
+        index = self.build_index_from_metadata("result-data-sample")
```

Comment:
You missed this one.

Reply:
that was in a different class and only one, so I avoided making changes there.